easypost/__init__.py

```diff
@@ -733,8 +733,7 @@ def get_smartrates(self):
         url = "%s/%s" % (self.instance_url(), "smartrate")
 
         response, api_key = requestor.request('get', url)
-        self.refresh_from(response, api_key)
-        return self
```

Comment:
Core of the change, return response not self, seems legit

Reply:
Right